tests/integration/test_main.py

```diff
@@ -456,23 +456,19 @@ def test_successful_parse(self, tmp_path):
         # GIVEN
         record_types = [
             "ALLOCATION",
+            "ALLOCATION_WITH_NATIVE",
```

Comment:
`ALLOCATION_WITH_NATIVE` is listed twice (both first and last line of the diff hunk)